<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s</class>
 <widget class="QWidget" name="s">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GeoAtlas Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1032</width>
        <height>282</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>GeoAtlas Settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="username_label">
               <property name="text">
                <string>Username</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="username"/>
             </item>
             <item>
              <widget class="QLabel" name="password_label">
               <property name="text">
                <string>Password</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="password">
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="inputMask">
                <string notr="true"/>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="role_label">
               <property name="text">
                <string>Role (Can be seen on the GeoAtlasLive website in the top right corner)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="role"/>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Custom Report style</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="1" column="0">
                 <widget class="QLineEdit" name="custom_qpt_file">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QPushButton" name="browseLocalFileButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Select qpt-file</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="use_custom_file">
                  <property name="text">
                   <string>Use Custom Report Style</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
